person.rb

```diff
@@ -0,0 +1,21 @@
+# the person class
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name: 'Unkown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] Your initialize function is missing an id, every new person created should have an id attached to it, see the code snippet below for guidance :+1: 
- code snippet
```ruby
@id = Random.rand(1..1000)
```